ci-operator/step-registry/openstack/provision/dpdk-networknodepolicy/openstack-provision-dpdk-networknodepolicy-commands.sh

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+export OS_CLIENT_CONFIG_FILE="${SHARED_DIR}/clouds.yaml"
+
+# For disconnected or otherwise unreachable environments, we want to
+# have steps use an HTTP(S) proxy to reach the API server. This proxy
+# configuration file should export HTTP_PROXY, HTTPS_PROXY, and NO_PROXY
+# environment variables, as well as their lowercase equivalents (note
+# that libcurl doesn't recognize the uppercase variables).
+if test -f "${SHARED_DIR}/proxy-conf.sh"
+then
+	# shellcheck disable=SC1090
+	source "${SHARED_DIR}/proxy-conf.sh"
+fi
+
+oc patch sriovoperatorconfig default --type=merge -n openshift-sriov-network-operator --patch '{ "spec": { "enableOperatorWebhook": false } }'
+sleep 5
+
+DPDK_NETWORKNODEPOLICY=$(
+    oc create -f - -o jsonpath='{.metadata.name}' <<EOF
+apiVersion: sriovnetwork.openshift.io/v1
+kind: SriovNetworkNodePolicy
+metadata:
+  name: dpdk1
+  namespace: openshift-sriov-network-operator
+spec:
+  deviceType: vfio-pci
+  nicSelector:
+    rootDevices:
```

Comment:
using `rootDevices` will be highly prolematic since @SchSeba is working on https://github.com/k8snetworkplumbingwg/sriov-network-device-plugin/pull/410.

He mentioned that we'll probably add `pciAddress` to the `nicSelector` at some point; which would help us.

From a UX standpoint, it's not super great as customers will have to figure out what PCI device to configure, but this is it for now.